cosmos/providers/databricks/notebook.py

```diff
@@ -193,6 +195,9 @@ def launch_notebook_job(self):
                 "base_parameters": {"source": self.source},
             },
             "libraries": self.notebook_packages,
+            "timeout_seconds": int(self.execution_timeout.total_seconds())
+            if self.execution_timeout
+            else 0,
```

Comment:
Would it make any sense to subtract the time the AIrflow task took to execute until this point - so we avoid the scenario we discussed:
- Airflow task: times out after 100s, but it spends 10s for setup
- Databricks job: takes 100s, receives timeout of 100s

This would lead the Airflow task would take 110s and fail, but the Databricks job would have been run successfully.

Reply:
I believe it should not take 10s for setup but ideally a few microseconds. Also, I don't know why but not totally convinced in having this manipulation here and the user would be confused as to why is the Databricks task timing out before the specified timeout value. As per Airflow, the task might fail at the first attempt due to timeout, but when it retries it marks the latest status of the task in the retried attempt.